tests/test_openreac.py

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
You are a bit light regarding tests, can you add test for the open reac API itself and not just for parameters ?